### PR TITLE
fix: Fixed rexnet1_3x and rexnet1_5x and updated checkpoints

### PR DESCRIPTION
**Pull request by frgfm**
This PR introduces the following modifications:
- fixes building of `rexnet1_3x` and `rexnet1_5x` (they were actually building a `rexnet1_0x` because of a typo)
- adds new features to training scipt: non-square image size, resize mode, updated augmentations, improved sample plotting, added gradient accumulation option, added an option to analyze training set image sizes and added a feature of Holocron to plot samples in the training set that yielded the highest loss
- retrained all rexnets & updated checkpoints
- updated the csv with performance
- fixed the resizing in API & demo

Any feedback is welcome!

### CODE REVIEW

**Comment by MateoLostanlen** [comment]
Hi @frgfm just a question did you use [readme parameters](https://github.com/pyronear/pyro-vision/tree/master/references/classification) for all models ? If not, it would be nice to add them to the performance table to keep track

**Comment by frgfm** [comment]
@MateoLostanlen I kept everything, but the args are not always the same from one model to the other. So we'll have to think about how to keep track of that & share it

**Comment by frgfm** [comment]
I just updated the readme instruction to display the command I use most often :+1:
